firmware/lib/config/config.h.template

```diff
@@ -17,6 +17,7 @@
 //#define WEB_DATA_WIDGET_URL "" // Use this to make your own widgets using an API/Webdata source
 //#define WEB_DATA_STOCK_WIDGET_URL "http://<insert host here>/stocks.php?stocks=SPY,VT,GOOG,TSLA,GME" // Use this as an alternative to the stock ticker widget
 #define WIDGET_CYCLE_DELAY 0 // Automatically cycle widgets every X ms, set to 0 to disable
+#define LOC_EN // Language selection for Month and Weekday - values are LOC_EN, LOC_DE, LOC_FR
```

Comment:
How about instead of three define vars we create one with a value?
#define LOCALE en

#if LOCALE de
...
#elif LOCALE fr
...
#else
...
#endif

It will be much clearer for people adding or using future languages.

Reply:
I like this approach @ericthelin, just much more user friendly.  If someone can make this change I'll merge it afterward.

Reply:
To make this work, we need an enum with the languages, right?

enum locales {
    de, en, fr
};

Otherwise de, en, fr are probably not defined...

Reply:
enums are not enough.

I think we can only compare it with other definitions, e.g.

```
#define EN 0
#define DE 1
#define FR 2

#define LOCALE FR

#if LOCALE==DE
...
#elif LOCALE==FR
...
#else
...
#endif
```

Reply:
Not sure if "define EN/DE/FR might interfere with anything and we should add a prefix like "LOC_"?

`#define LOC_EN 0`

But this does not look that nice for the user

`#define LOCALE LOC_EN`